includes/blocks/safe-svg/frontend.scss

```diff
@@ -1,15 +1,33 @@
-.safe-svg-cover {
-    text-align: center;
-
-    .safe-svg-inside {
-        display: inline-block;
-        max-width: 100%;
-    }
-
-    svg {
-        height: 100%;
-        max-height: 100%;
-        max-width: 100%;
-        width: 100%;
-    }
+.wp-block-safe-svg-svg-icon	{
+	svg {
+		display: block;
```

Comment:
If this is here to prevent the whitespace after the SVG, can we use the `vertical-align: middle` method instead?